go/billing/settings.py

```diff
@@ -43,7 +43,7 @@
 STATEMENT_CONTACT_DETAILS = getattr(
     settings, 'BILLING_STATEMENT_CONTACT_DETAILS', {
```

Comment:
All our other billing settings use `BILLING_` as a prefix. It is only the new credit notification settings that use `GO_`. If we are to change the prefix to `GO_`, then the other settings should change too.